packages/react-examples/src/react/DetailsList/DetailsList.KeyboardAccessibleResizeAndReorder.Example.tsx

```diff
@@ -0,0 +1,320 @@
+        if (draggedItem) {
+          insertBeforeItem(item);
+        }
+      },
+      onDragStart: (item?: any, itemIndex?: number, selectedItems?: any[], event?: MouseEvent) => {
+        setDraggedItem(item);
+        setDraggedIndex(itemIndex!);
+      },
+      onDragEnd: (item?: any, event?: DragEvent) => {
+        setDraggedItem(undefined);
+        setDraggedIndex(-1);
+      },
+    };
+  }, [draggedItem, insertBeforeItem]);
+
+  const dragDropEvents = getDragDropEvents();
+
+  const resizeDialogContentProps = {
+    type: DialogType.normal,
+    title: 'Resize Column',
+    closeButtonAriaLabel: 'Close',
+    subText: 'Enter desired column width pixels:',
+  };
+
+  const reorderDialogContentProps = {
+    type: DialogType.normal,
+    title: 'Reorder Column',
+    closeButtonAriaLabel: 'Close',
+    subText: 'Enter which column to move this to (use 1-based indexing):',
+  };
+
+  const modalProps = {
+    titleAriaId: 'Dialog',
+    subtitleAriaId: 'Dialog sub',
+    isBlocking: false,
+    styles: dialogStyles,
+  };
+
+  return (
+    <div>
+      <div className={controlWrapperClass}>
+        <Toggle
+          label="Enable column reorder"
+          checked={isColumnReorderEnabled}
+          onChange={onChangeColumnReorderEnabled}
+          onText="Enabled"
+          offText="Disabled"
+          styles={togglesStyles}
+        />
+        <TextField
+          label="Number of left frozen columns"
+          onGetErrorMessage={validateNumber}
+          value={frozenColumnCountFromStart}
+          onChange={onChangeStartCountText}
+          styles={textFieldStyles}
+        />
+        <TextField
+          label="Number of right frozen columns"
+          onGetErrorMessage={validateNumber}
+          value={frozenColumnCountFromEnd}
+          onChange={onChangeEndCountText}
+          styles={textFieldStyles}
+        />
+      </div>
+      <MarqueeSelection selection={selection}>
+        <DetailsList
+          componentRef={detailsListRef}
+          setKey="items"
+          items={sortedItems}
+          columns={columns}
+          selection={selection}
+          selectionPreservedOnEmptyClick={true}
+          onItemInvoked={onItemInvoked}
+          onRenderItemColumn={onRenderItemColumn}
+          dragDropEvents={dragDropEvents}
+          columnReorderOptions={isColumnReorderEnabled ? getColumnReorderOptions() : undefined}
+          ariaLabelForSelectionColumn="Toggle selection"
+          ariaLabelForSelectAllCheckbox="Toggle selection for all items"
+          checkButtonAriaLabel="select row"
+        />
+      </MarqueeSelection>
+      {contextualMenuProps && <ContextualMenu {...contextualMenuProps} />}
+
+      <Dialog
+        hidden={isDialogHidden}
+        onDismiss={hideDialog}
+        dialogContentProps={clickHandler.current === RESIZE ? resizeDialogContentProps : reorderDialogContentProps}
+        modalProps={modalProps}
+      >
+        <TextField componentRef={textfieldRef} />
```

Comment:
The textfield needs an accessible name (probably doesn't need to be visible, since the dialog title essentially functions as the visible label)

Reply:
Added here: 6333b10